src/function.js

```diff
@@ -0,0 +1,34 @@
+const apply = (listTask, container) => {
+  const sortedTodos = listTask.list.sort();
+  container.innerHTML = '';
+  let todosHtml = '';
+  sortedTodos.forEach((todo) => {
+    todosHtml += `  <div class="item">
+                              <div>
+                                  <input id="${todo.id}" type="checkbox" class="checkbox" />
+                                  <input id="${todo.index}" type="text" class="todo-edit" value="${todo.description}" />
+                            </div>
+                        <i id="${todo.index}" class="remove fas fa-trash"></i>
+                    </div>
+          `;
+  });
+
+  container.innerHTML = todosHtml;
+
+  const removeBtns = document.querySelectorAll('.remove');
+  removeBtns.forEach((btn) => {
+    btn.addEventListener('click', (e) => {
+      const element = btn.parentNode;
+      listTask.RemoveTask(Number(e.target.parentNode.id));
+      element.remove();
+    });
+  });
```

Comment:
- [ ] **Required:** This function still behave in an abnormal manner.
![localStorage not working](https://user-images.githubusercontent.com/27709832/150413633-ae848d7c-c0e9-4005-9f7c-05faf3f77464.gif)

Please fix this functionality by looking at all remove functionality as well as the how you get list items to display.